pagopa/pn-frontend
# fix(PN-11950): add current organization in institutions when not present

## Short description
Add current organization in institutions when not present

## List of changes proposed in this pull request
- Moved logic in actions file for PA from App.tsx
- Added current institution in institutions list when it's not present

## How to test
Login as PA; check current institution is present in list of institutions drawer.
